srcpkgs/you-get/template

```diff
@@ -1,17 +1,17 @@
 # Template file for 'you-get'
 pkgname=you-get
-version=0.4.1410
-revision=1
+version=0.4.1423
+revision=2
```

Comment:
revision should be reset to 1 on version updated.

Reply:
done.